tests/filecheck/mlir-conversion/with-mlir/dialects/transform/transform_types.mlir

```diff
@@ -1,6 +1,10 @@
 // RUN: xdsl-opt %s | xdsl-opt | mlir-opt --allow-unregistered-dialect | filecheck %s
 
 builtin.module attributes  {"transform.with_named_sequence"} {
+  "transform.named_sequence"() <{arg_attrs = [{transform.readonly}], function_type = (!transform.any_op) -> (), sym_name = "foo"}> ({
+  ^bb0(%arg0: !transform.any_op):
+    "transform.yield"() : () -> ()
+  }) : () -> ()
```

Comment:
Please use custom syntax in tests when possible, it makes them more readable

Reply:
I don't really see where custom syntax can be used in this case. are there any particular operations you're referring to?